### PR TITLE
Add new upcoming events [Fixes #5221]

### LINKED ISSUE #5221: Add 2022 Ethereum events
**Is your feature request related to a problem? Please describe.**

More events have been added to this sheet:
https://docs.google.com/spreadsheets/d/1NEu_FCc1hnGAuRgPmbXXpf0h2lCrCOlMKbbFEqgkVDQ/htmlview

**Describe the solution you'd like**

Add them to https://ethereum.org/en/community/events/

**Describe alternatives you've considered**

Should we also add a link to this spreadsheet? Seems like it's updated fairly reguarly.

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by vluna**
## Description
- Add events from [the provided spreadsheet](https://docs.google.com/spreadsheets/d/1NEu_FCc1hnGAuRgPmbXXpf0h2lCrCOlMKbbFEqgkVDQ/htmlview)
- Update component to render 10 events at a time with "Load more" button

## Related Issue
#5221

### CODE REVIEW

**Review by minimalsm — CHANGES_REQUESTED** [review]
Nice! Thanks @vluna.

A few thoughts:

If we don't have a description, we should write a simple one. 
e.g. ETHWarsaw is a hackathon in Warsaw, Poland, expected in Q4 2022.

I don't think we should link out to dead meetup.com meetings. Do they have an alternative? E.g. Twitter or if they are part of ETHGlobal we could just link there?

If something is TBD, we don't currently have a good way to show this and spanning the event across the whole quarter is confusing to users.
- We could remove ETHWarsaw, ETHKyiv, Liscon, EthLisbon, ETHBogota, DevCon until they get official dates IMO.
- We could refactor the component to better support this.

_Less of a priority but this list is super long now. I think we should paginate or add 'load more' functionality to reduce cognitive load here._

File: src/data/community-events.json [hunk 1]
```diff
@@ -53,15 +116,105 @@
...
+    "title": "NFT Berlin",
+    "to": "https://mirror.xyz/0x37AEe71F739FB44D9240CB14D41bE16Df767F79e/1vBwQXFYHpnCyk9R5J_dl77oOzxetRfF0Li0nE5guZA",
+    "sponsor": null,
+    "location": "Berlin, DE",
+    "description": "NFTBERLIN is the people’s platform. We see ourselves as the public good of the metaverse, accelerating the democratisation of culture through creativity and technology.",
+    "startDate": "2022-06-01",
+    "endDate": "2022-06-30"
+  },
+  {
+    "title": "ETH SF",
```
[the remark this answers is not in the document]
Comment (vluna) [reply src/data/community-events.json:156]: Yup, it seems like it.

**Comment by vluna** [comment]
@minimalsm Applied your suggestions. Let me know what you think.

**Review by wackerow — APPROVED** [review]
Aw yeah, great to see all these events getting listed. Thanks @vluna!

@minimalsm I think using `isSecondary` on that button would be more fitting, but we can debate that separately... gonna bring this in

**Comment by wackerow** [comment]
Build is showing as failed because it ran out of memory:
```
Your Gatsby build's memory consumption exceeded the limits allowed in your plan. For more details, see https://gatsby.dev/memory.
```

I cleared cache and rebuilt it and it worked fine, just doesn't seem to be updating here.

**Pull request merged**